meetings/2023-05-03.md

```diff
@@ -0,0 +1,65 @@
+# Node.js Technical Steering Committee (TSC) Meeting 2023-05-03
+
+## Links
+
+* **Recording**: <https://www.youtube.com/watch?v=8_iAbWu7KLs>
+* **GitHub Issue**: <https://github.com/nodejs/TSC/issues/1381>
+
+## Present
+
+* Danielle Adams @danielleadams (voting member)
+* Joyee Cheung @joyeecheung (voting member)
+* Matteo Collina @mcollina (voting member)
+* Michael Dawson @mhdawson (voting member)
+* Moshe Atlow @MoLow (voting member)
+* Darshan Sen @RaisinTen (voting member)
+
+## Agenda
+
+### Announcements
+
+* OpenJS Conference next week, including collaborator summit
+
+### CPC and Board Meeting Updates
+
+*Extracted from **tsc-agenda** labeled issues and pull requests from the **nodejs org** prior to the meeting.
+
+* Board updates
+  * CPC proposing some changes to foundation bylaws in respect to
+  * how community directors are elected. Important that people
+  * take a look to make sure they don’t have any concerns.
+  * <https://github.com/openjs-foundation/cross-project-council/pull/1055>
+
+### nodejs/TSC
+
+* Adding the OpenJS Foundation staff as a billing manager for nodejs-private [#1376](https://github.com/nodejs/TSC/issues/1376)
+* No objections from the 6 TSC members at the meeting, will make note in issue and the remove from agenda.
+
+* Opportunity: Grace Hopper Open Source Day (virtual) Sept 22, 2023 [#1372](https://github.com/nodejs/TSC/issues/1372)
+  * Believe past submission date so can take it off the agenda
+
+* 2023 TSC Elections [#1383](https://github.com/nodejs/TSC/issues/1382)
+  * FYI on upcoming election for Chair and Vice-chair
+
+* openjs-foundation/cross-project-council [#397](https://github.com/openjs-foundation/cross-project-council/issues/397)
+  * Travel fund was migrated from Node.js project to CPC/Foundation level so all projects could use the travel fund
+  * Has enabled lots of things
+  * Today you must make public request with info on conference info, travel dates etc.
+  * Question being discussed about privacy. The requesting info might be problematic.
+  * Discussion is that requests may be made private versus public
+  * If anybody has concerns about requests being private versus public please jump into the discussion.
+  * Danielle, have not asked for funds due to public nature of it so has been a barrier in that sense.
+
+## Strategic Initiatives
```

Comment:
```suggestion
## Strategic Initiatives

* ShadowRealm (Chengzhong) - Working on the finalization order of base objects and their nested fields when the shadow realm is being finalized - <https://github.com/nodejs/node/pull/47688>.
```

from @legendecas's comment in https://github.com/nodejs/TSC/issues/1381#issuecomment-1532988149